examples/Mp3PlayerDemo/Mp3PlayerDemo.ino

```diff
@@ -45,9 +54,17 @@
 #include "SampleMp3.h"
 
 // Wiring of VS1053 board (SPI connected in a standard way)
+#if defined(nodemcuv2)
```

Comment:
This is fine in the example, but in productive code I would rather use a generic check.
I mean `#ifdef ARDUINO_ARCH_ESP32` to check whether ESP32 and `#ifdef ARDUINO_ARCH_ESP8266` for ESP8266. 
So a generic check for platform rather than a specific board. As this implementation would work let's say for `nodemcuv2` but very similar one `d1` (Wemos D1) which is also esp8266 would be probably not handled.
But it's fine here for an example.

Reply:
I think that general-purpose check is better